README.md

```diff
@@ -485,23 +545,25 @@ There are now three options for enabling docker DNS resolution from within the w
 
 2. If `LOCAL_TLD` is set (e.g. to local) write `/etc/dnsmasq.conf` to use `127.0.0.11` for that TLD. Also, write `/etc/resolv.conf` to search `LOCAL_TLD` so that containers can access the addresses of the services without having to know to append `.local` to match the rule in `/etc/dnsmasq.conf`. This will require aliases with the TLD in each of the containers that need to be accessible from within the wireguard network. An alias should look something like this (e.g. for `LOCAL_TLD=ghost`).
 
-```
-filebot:
-...
-  networks:
-    default:
-      aliases:
-        - filebot.ghost
-```
+   ```yaml
+   filebot:
+   ---
```

Comment:
The `...` here is supposed to be an ellipses to indicate that there's probably a lot of stuff between `filebot` and when you're specifying the network configuration. What if instead we change it to

```yaml
  filebot:
    image: ghostserverd/filebot:4.9.x
    container_name: filebot
    restart: always
    networks:
      default:
        aliases:
          - filebot.ghost
```

Also are there supposed to be three leading spaces? I'd prefer two or four.

Reply:
Somehow I didn't catch that my IDE changed all dots to dashes... I've fixed this as well as spaces (even if linter intentionally changes them to 3 all the time :/)